predicate/cli/__main__.py

```diff
@@ -14,60 +18,93 @@ def main():
 @main.command()
 @click.argument("policy-file")
 def export(policy_file):
-    module = run_path(policy_file)
+    """
+    Export to YAML compatible policy.
+    """
+    with open(policy_file, 'r', encoding="utf-8") as file:
+        class_name = get_classname(file.read())
```

Comment:
Since get_classname more or less only works on Python files. What do you think about moving the file read logic to inside there instead to get rid of the extra ident level in all callees?

Reply:
Agree. Resolved in the following commit.